protocol/v2/ssv/validator/committee.go

```diff
@@ -254,22 +222,22 @@ func (c *Committee) ProcessMessage(logger *zap.Logger, msg *queue.SSVMessage) er
 		}
 
 		if pSigMessages.Type == spectypes.PostConsensusPartialSig {
-			c.mtx.Lock()
-			runner, exists := c.Runners[pSigMessages.Slot]
-			c.mtx.Unlock()
+			c.mtx.RLock()
+			r, exists := c.Runners[pSigMessages.Slot]
+			c.mtx.RUnlock()
 			if !exists {
-				return errors.New("no runner found for message's slot")
+				return errors.New(fmt.Sprintf("no runner found for post consensus sig message's slot: %d", pSigMessages.Slot))
```

Comment:
```suggestion
				return fmt.Errorf("no runner found for post consensus sig message's slot %d", pSigMessages.Slot)
```

Reply:
Applied in https://github.com/ssvlabs/ssv/pull/1784/commits/483c2ed41d9b789251d57e9d6ce4f3275e3f02c1